Pythonwin/pywin/Demos/guidemo.py

```diff
@@ -26,30 +26,28 @@
 
 
 def demo():
-    try:
-        # seeif I can locate the demo files.
-        import fontdemo
-    except ImportError:
-        # else put the demos direectory on the path (if not already)
-        try:
-            instPath = regutil.GetRegistryDefaultValue(
-                regutil.BuildDefaultPythonKey() + "\\InstallPath"
-            )
-        except win32api.error:
-            print(
-                "The InstallPath can not be located, and the Demos directory is not on the path"
-            )
-            instPath = "."
-
-        demosDir = win32ui.FullPath(instPath + "\\Demos")
-        for path in sys.path:
-            if win32ui.FullPath(path) == demosDir:
-                break
-        else:
-            sys.path.append(demosDir)
-        import fontdemo
+    # try:
+    #     # seeif I can locate the demo files.
+    #     import fontdemo
+    # except ImportError:
+    #     # else put the demos direectory on the path (if not already)
+    #     try:
+    #         instPath = regutil.GetRegistryDefaultValue(
+    #             regutil.BuildDefaultPythonKey() + "\\InstallPath"
+    #         )
+    #     except win32api.error:
+    #         print(
+    #             "The InstallPath can not be located, and the Demos directory is not on the path"
+    #         )
+    #         instPath = "."
 
-    import sys
+    #     demosDir = win32ui.FullPath(instPath + "\\Demos")
+    #     for path in sys.path:
+    #         if win32ui.FullPath(path) == demosDir:
+    #             break
+    #     else:
+    #         sys.path.append(demosDir)
+    #     import fontdemo
```

Comment:
The demo was already commented out above. TBD if fontdemo should just be removed entirely from here or added back in. It's possible that #2101 fixed w/e issue fontdemo had.

Reply:
Fixed by #2345